docs/source/generic/rest/pagination.rst

```diff
@@ -40,19 +38,36 @@ In de response zijn de volgende elementen te vinden:
 Met de velden ``_links.next`` en ``_links.previous`` zijn respectievelijk de volgende en vorige pagina op te vragen.
 Meer algemeen kan pagina `n` worden opgevraagd met :samp:`?page={n}`.
 Paginanummers beginnen bij één, niet nul.
+De links ``next`` en ``previous`` ontbreken op de eerste, resp. laatste, pagina.
 
 In het object ``page`` zijn de volgende velden opgenomen:
 
-* ``page.number``: Het huidige paginanummer.
-* ``page.size``: De grootte van een pagina.
-* ``page.totalElements``: Het aantal objecten in de (gefilterde) resultaat set.
-* ``page.totalPages``: Het aantal paginas in de (gefilterde) resultaat set.
+* ``page.number``: het huidige paginanummer.
+* ``page.size``: de grootte van een pagina.
 
-`page.totalPages` en `page.totalElements` worden alleen teruggegeven als de `_count` parameter wordt gebruikt.
+Wordt ``_count=true`` meegegeven, dat bevat de response tevens
```

Comment:
```suggestion
Wordt ``?_count=true`` meegegeven, dat bevat de response tevens:
```

Die `?` notatie wordt (bijna) overal gebruikt om te duiden dat het een query-string parameter is.